src/GraphQL.Primitives/GraphQL.Primitives.csproj

```diff
@@ -6,4 +6,7 @@
     <TargetFrameworks>netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Memory" Version="4.5.5" />
```

Comment:
Where is this needed?  This reference is fine with me, but on the other hand, there's probably no reason to spend time optimizing older frameworks.  If users want an optimized implementation, they can upgrade to a modern version of .NET.  (Note: I am greatly in favor of supporting old frameworks via .NET Standard.)

Reply:
The current implementation of the Hash class uses `System.Buffers` which gets pulled in by `System.Memory`. I think just referencing `System.Buffers` would be better here.